dayinfo.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import calendar_sk
+
+
+def formatter(data, month, day, prefix=''):
+    """Format data"""
+    if isinstance(data, list):
+        info = data[month-1][day-1]
+    else:
+        try:
+            info = data[str(month)][str(day)]
+        except KeyError:
+            info = None
+
+    if info is None:
```

Comment:
@mrshu On line 12, I can grab None from nameday list as there is no name set for that day